lib/ansible/modules/windows/win_chocolatey_facts.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Ansible Project
+# Copyright: (c) 2018, Simon Baerlocher <s.baerlocher@sbaerlocher.ch>
+# Copyright: (c) 2018, ITIGO AG <opensource@itigo.ch>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: win_chocolatey_facts
+version_added: '2.7'
+short_description: Create a facts collection for chocolatey
+description:
+   - This module shows information from chocolatey, such as installed packages, configuration, feature and sources.
+author:
+    - Simon Bärlocher (@sbaerlocher)
+    - ITIGO AG (@itigoag)
+'''
+
+EXAMPLES = r'''
+- name: Gather facts from chocolatey
+  win_chocolatey_facts:
+
+- name: Displays the Configuration
+  debug:
+    var: ansible_chocolatey.config
+
+- name: Displays the Feature
+  debug:
+    var: ansible_chocolatey.feature
+
+- name: Displays the Sources
+  debug:
+    var: ansible_chocolatey.sources
+
+- name: Displays the Packages
+  debug:
+    var: ansible_chocolatey.packages
+'''
+
+RETURN = r'''
+ansible_facts:
+    description: Detailed information about the chocolatey installation.
+    returned: always
+    type: complex
+    contains:
+        ansible_chocolatey:
+            description: Detailed information about the Chocolatey installation.
+            returned: infos when choco is found.
+            type: dict
+            contains:
+                config:
+                    description: Detailed information about stored the Configurations.
+                    returned: always
+                    type: dict
+                feature:
+                    description: Detailed information about enabled and disabled functions.
+                    returned: always
+                    type: dict
+                sources:
+                    description: Detailed information about the deposited Chocolatey sources.
```

Comment:
Should say this is a list of Chocolatey sources and then put the keys under the contains subkey, e.g.

```
sources:
  description: List of Chocolatey sources
  returned: always
  type: complex
  contains:
    admin_only:
    ....
```